packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js

```diff
@@ -200,7 +202,7 @@ const Mercury = WebexPlugin.extend({
         return this.webex.internal.feature.getFeature('developer', 'web-high-availability')
           .then((haMessagingEnabled) => {
             if (haMessagingEnabled) {
-              return this.webex.internal.device.fetchNewUrls(attemptWSUrl);
+              return this.webex.internal.device.refresh();
```

Comment:
Turns out nothing was using the URL returned back from `fetchNewUrls` so a refresh is all that is needed.